### ISSUE TITLE
Add support for UX410UQK

### ISSUE THREAD

**Issue by gobenavides** [dominiksalvet/asus-fan-control#21]
**Environment**
asus-fan-control version: 3.1.0
OS: Ubuntu 18.04.4

**Basic configuration**
* `sudo asus-fan-control get-temps`
```
50 55 60 62 65 68 72 75
```

* `sudo asus-fan-control model-info`
```
name UX410UQK
address 1335
temperatures '55 60 62 65 68 72 76 80'
tested false
```

**Additional context**
Tested in room at approximately 24 °C.
When idle, laptop at approximately 42 °C.

**Comment by dominiksalvet** [comment]
Hello @gobenavides and welcome to the asus-fan-control community! :wave: 

I will add the support for your laptop model in a moment. :rocket: 

I have a question, though. Does the `model-info` output, namely the `temperatures` section, contains the apostrophes (`'`) around the temperatures? If not, please update your post... Thank you. :heart:

**Comment by dominiksalvet** [comment]
All right, the support for your model has been added in [3.2.0](https://github.com/dominiksalvet/asus-fan-control/releases/tag/3.2.0) version.

Just run `sudo gitpack install github.com/dominiksalvet/asus-fan-control` and you are running the latest asus-fan-control version. Then, try to run `sudo asus-fan-control model-info` and now you should have different output than before adding support! :rocket:

BTW, can I reformat a bit your first post? Please, do not forget about my previous post and the apostrophes... :smile:

**Comment by gobenavides** [comment]
> Hello @gobenavides and welcome to the asus-fan-control community! wave
> 
> I will add the support for your laptop model in a moment. rocket
> 
> I have a question, though. Does the `model-info` output, namely the `temperatures` section, contains the apostrophes (`'`) around the temperatures? If not, please update your post... Thank you. heart

Thanks for your warm welcome.

It does contain the apostrophes.

**Comment by gobenavides** [comment]
> All right, the support for your model has been added in [3.2.0](https://github.com/dominiksalvet/asus-fan-control/releases/tag/3.2.0) version.
> 
> Just run `sudo gitpack install github.com/dominiksalvet/asus-fan-control` and you are running the latest asus-fan-control version. Then, try to run `sudo asus-fan-control model-info` and now you should have different output than before adding support! rocket
> 
> BTW, can I reformat a bit your first post? Please, do not forget about my previous post and the apostrophes... smile

Nice, it works :)

**Comment by dominiksalvet** [comment]
> > Hello @gobenavides and welcome to the asus-fan-control community! wave
> > I will add the support for your laptop model in a moment. rocket
> > I have a question, though. Does the `model-info` output, namely the `temperatures` section, contains the apostrophes (`'`) around the temperatures? If not, please update your post... Thank you. heart
> 
> Thanks for your warm welcome.
> 
> It does contain the apostrophes.

That's weird, it should not have contained the apostrophes. My output does not contain apostrophes (and I have Ubuntu 18.04.4 as well). Could you try a different shell? E.g., run `sh` command and then try `sudo asus-fan-control model-info` whether it contains apostrophes as well. If so, do you use the default terminal emulator in Ubuntu (`gnome-terminal`)?

**Comment by dominiksalvet** [comment]
> > All right, the support for your model has been added in [3.2.0](https://github.com/dominiksalvet/asus-fan-control/releases/tag/3.2.0) version.
> > Just run `sudo gitpack install github.com/dominiksalvet/asus-fan-control` and you are running the latest asus-fan-control version. Then, try to run `sudo asus-fan-control model-info` and now you should have different output than before adding support! rocket
> > BTW, can I reformat a bit your first post? Please, do not forget about my previous post and the apostrophes... smile
> 
> Nice, it works :)

**Thank you** so much for contributing to asus-fan-control. :heart: :rocket:

**Comment by gobenavides** [comment]
> > > Hello @gobenavides and welcome to the asus-fan-control community! wave
> > > I will add the support for your laptop model in a moment. rocket
> > > I have a question, though. Does the `model-info` output, namely the `temperatures` section, contains the apostrophes (`'`) around the temperatures? If not, please update your post... Thank you. heart
> > 
> > 
> > Thanks for your warm welcome.
> > It does contain the apostrophes.
> 
> That's weird, it should not have contained the apostrophes. My output does not contain apostrophes (and I have Ubuntu 18.04.4 as well). Could you try a different shell? E.g., run `sh` command and then try `sudo asus-fan-control model-info` whether it contains apostrophes as well. If so, do you use the default terminal emulator in Ubuntu (`gnome-terminal`)?

Ok, I downgraded asus-fan-control via `sudo gitpack install github.com/dominiksalvet/asus-fan-control=3.1.0`, ran `sh`, and—inside the environment—tried `sudo asus-fan-control model-info` obtaining the same output.

**Comment by gobenavides** [comment]
> > > > Hello @gobenavides and welcome to the asus-fan-control community! wave
> > > > I will add the support for your laptop model in a moment. rocket
> > > > I have a question, though. Does the `model-info` output, namely the `temperatures` section, contains the apostrophes (`'`) around the temperatures? If not, please update your post... Thank you. heart
> > > 
> > > 
> > > Thanks for your warm welcome.
> > > It does contain the apostrophes.
> > 
> > 
> > That's weird, it should not have contained the apostrophes. My output does not contain apostrophes (and I have Ubuntu 18.04.4 as well). Could you try a different shell? E.g., run `sh` command and then try `sudo asus-fan-control model-info` whether it contains apostrophes as well. If so, do you use the default terminal emulator in Ubuntu (`gnome-terminal`)?
> 
> Ok, I downgraded asus-fan-control via `sudo gitpack install github.com/dominiksalvet/asus-fan-control=3.1.0`, ran `sh`, and—inside the environment—tried `sudo asus-fan-control model-info` obtaining the same output. I do use `gnome-terminal`.

By the way, in version 3.2.0 the apostrophes no longer appear after running `sudo asus-fan-control model-info`.

**Comment by dominiksalvet** [comment]
@gobenavides Hey man, thank you so much for the feedback. I have identified where the apostrophes come from... I will fix it soon. I will leave the commit hash here once I fix it for your inspection... :mag: 

Thank you so much for everything! :smile: One added laptop model and two bugs! Asus-fan-control has been elevated! :rocket:

**Comment by dominiksalvet** [comment]
@gobenavides All right, so it has been fixed in baf6892d6555211b77894d8f696d7d24349661f5 and it was actually more serious problem. Maybe I will release a new version with this fix soon. However, tested models should not be affected at all.

Funny how many bugs [3.0.0](https://github.com/dominiksalvet/asus-fan-control/releases/tag/3.0.0) version has brought... Fortunately, we have people like you! :heart: :rocket: 

BTW I have updated your first post by removing the apostrophes not to confuse new people. :+1:
